pocs/scheduler/constraint.py

```diff
@@ -159,3 +165,160 @@ def get_score(self, time, observer, observation, **kwargs):
 
     def __str__(self):
         return "Moon Avoidance"
+
+
+class Horizon(BaseConstraint):
+
+    """ Implements horizon and obstruction limits"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.obstruction_points = []
+
+        print("Horizon.__init__")
```

Comment:
Need to remove all `print` statements and use the `self.logger.debug` (or `.info`, `.warning`, etc) system.